src/store/callViewStore.js

```diff
@@ -26,6 +26,8 @@ import {
 	CONVERSATION,
 } from '../constants'
 
+const LRU = require('lru-cache')
```

Comment:
normally we use import? not require?

Reply:
The library does not support `import`, only `require` :shrug: